lib/storage.rb

```diff
@@ -0,0 +1,121 @@
+require_relative 'book'
+require_relative 'label'
+require_relative 'author'
+require 'json'
+
+class Storage
+  def init_books
```

Comment:
- [ ] Kindy ensure you initialize a JSON file with an empty array  for your book author and label so that when a user deletes the file, it is automatically created when the app is initialized: I experience your app refusing to start after I deleted one of your son files:
![image](https://user-images.githubusercontent.com/61408860/184171948-cba21df7-06db-42ff-b9f3-8cd5080bc795.png)

Reply:
HI, I did not face this kind of issue...as I show in the video...Here is the video link (https://www.loom.com/share/36fe5126a991411b9522583345d1cf8d)